### ISSUE TITLE
LUDE Web Application Error

### ISSUE THREAD

**Issue by ShubhamTalware** [Capigol/LUDe_v2#5]
The web application is not working.

Page is not loading even after multiple trials.

**Comment by Capigol** [comment]
Dear Shubham Talware! I hope you are well!

LUDe had run out of memory and needed to be restarted. We have already solved the problem.
Please keep in mind that this webapp runs on the streamlit server and it has limited memory. I don't think it will be possible to run the app for more than 30 query molecules. If you need to generate decoys for more molecules, I recommend you to download the github repository and run the script on your PC. I'm sending you some tips so you can run LUDe on your PC (sorry about my english...).


1) Download the LUDe github:
https://github.com/Capigol/LUDe_v2
(you must click on the  green "code" button and then download ZIP.

2) if you don't have anaconda installed, install it:
https://docs.anaconda.com/free/anaconda/install/mac-os/

3) Open the Anaconda Prompt (like a Linux terminal) and create a new conda environment:
you will have to write the following in the Anaconda Prompt:

conda create -n LUDe python=3.9

3) Activate the conda environment:
write in the Promt: 

activate LUDe

4) Install the required packages:
run the following lines of code one by one:

python -m ensurepip --upgrade
pip install streamlit
pip install rdkit-pypi
pip install MolVS
pip install openbabel-wheel

5) Search for the directory where you have download and unzipped the LUDe github. For example, in my computer (windows) it is 
"C:\Users\adm\Downloads\LUDe_v2-main"

6) In the anaconda prompt write:
cd "C:\Users\adm\Downloads\LUDe_v2-main "
replace what's in quotes with your LUDe directory

7) Run LUDe!
Write in the prompt: 

streamlit run LUDe_streamlit_v2.py

If all of the above worked, LUDe should open in your browser. I trust that you can achieve it. If you have any questions, write to me again.

If you didn't succeed and you need the decoys urgently, you can send me the queries and I will run them on my PC.

Best regards!

**Comment by ShubhamTalware** [comment]
Thank you so much for your kind reply.

I'll go through it once, and revert back to you in case of any queries.

*Regards,*
*Shubham Talware.*

**Comment by ShubhamTalware** [comment]
I wanted to inquire whether we can use only 100 compounds at one time for
decoy generation?

Because it gave an error beyond that.

*Regards,*
*Shubham Talware.*

**Comment by Capigol** [comment]
Yes, in the desktop version you can run more than 100 molecules.  The maximum will depend on the characteristics of your computer.  The streamlit (webapp) version does not support more than 20 molecules due to memory limits.
Here is my email for other questions: lucasalberca@gmail.com

**Comment by ShubhamTalware** [comment]
Please find the attached file.

I have shared the image of my screen below.

I'm still getting this recurrent error even after so many tries.

Can you help me out with this?

Thanks.

*Regards,*

*Shubham Talware.*

**Comment by Capigol** [comment]
Ok! I am on holidays for a few days, but I will try as soon as possible.
Maybe today or tomorrow.
Sorry for the delay!

**Comment by Capigol** [comment]
I can't find the attachment on github. I can't see the image of your screen either. Could you send me both files by email? 
lucasalberca@gmail.com
